cores/migrations/20240628024728_AddSummaryToPageSnapShot.up.sql

```diff
@@ -0,0 +1,2 @@
+-- Add migration script here
+ALTER TABLE PageSnapshot ADD COLUMN summary TEXT DEFAULT '' NOT NULL;
```

Comment:
Ahh we probably want to remove this migration, since we are saving the summary to `markdown` right?

I think to do that nicely you can do `sqlx migrate revert` TWICE, since there are two migrations to revert. Then you can delete this up and the down and re-run migrate to add the summary to markdown!